### ISSUE TITLE
content of <content> contentious

### ISSUE THREAD

**Issue by sydb** [TEIC/TEI#1716]
Per https://github.com/TEIC/TEI/issues/1459#issuecomment-345356270, creating a new ticket for the problematic content of `<content>`.

In P5 the content model of `<content>` is currently defined as
~~~~~xml
    <alternate minOccurs="0" maxOccurs="unbounded">  
      <anyElement require="http://relaxng.org/ns/structure/1.0"/>
      <classRef key="model.contentPart" />
    </alternate>
~~~~~
This allows intermingling of RELAX NG and PureODD elements. My understanding is this was not what was intended, and should not be allowed. Thus the content model for `<content>` should be
~~~~~xml
    <alternate>
      <anyElement require="http://relaxng.org/ns/structure/1.0" minOccurs="1" maxOccurs="unbounded"/>
      <classRef key="model.contentPart" minOccurs="0" maxOccurs="unbounded"/>
    </alternate>
~~~~~~
HOWEVER, I just discovered that the P5 build process does not correctly propegate the ordinality of the `<anyElement>`. That is, it produces
~~~~~
  ( anyElement-content | model.contentPart* )
~~~~~
instead of
~~~~~
  ( anyElement-content+ | model.contentPart* )
~~~~~
I.e., the ‘+’ (or `<rng:oneOrMore>`) is missing. So I claim implementation of this ticket is blocked by an as-of-yet non-existent Stylesheets ticket.

**Comment by sydb** [comment]
Fixed at e46f655. Let’s see what Mr. Jenkins thinks before closing, though.
